modules/ROOT/pages/syntax-quick-reference.adoc

```diff
@@ -0,0 +1,1292 @@
+[#ex-csv-formal]
+----
+include::tables:example$data.adoc[tag=csv]
+----
+
+.View result of <<ex-csv-formal>>
+[%collapsible.result]
+====
+include::tables:example$data.adoc[tag=csv]
+====
+
+.Table from CSV data in file
+[#ex-csv-include]
+----
+include::tables:example$data.adoc[tag=i-csv]
+----
+
+.Table from DSV data using shorthand
+[#ex-dsv]
+----
+include::tables:example$data.adoc[tag=s-dsv]
+----
+
+.View result of <<ex-dsv>>
+[%collapsible.result]
+====
+include::tables:example$data.adoc[tag=s-dsv]
+====
+
+.Table with formatted, aligned and merged cells
+[#ex-cell-format]
+----
+include::tables:example$cell.adoc[tag=b-spec]
+----
+
+.View result of <<ex-cell-format>>
+[%collapsible.result]
+====
+include::tables:example$cell.adoc[tag=b-spec]
+====
+
+== Keyboard, button, and menu macros
+
+IMPORTANT: You must set the `experimental` attribute in the document header to enable these macros.
+
+.Keyboard macro
+[#ex-kbd]
+----
+include::macros:example$ui.adoc[tag=qr-key]
+----
+
+.View result of <<ex-kbd>>
+[%collapsible.result]
+====
+include::macros:example$ui.adoc[tag=qr-key]
+====
+
+.Menu macro
+[#ex-menu]
+----
+include::macros:example$ui.adoc[tag=menu]
+----
+
+.View result of <<ex-menu>>
+[%collapsible.result]
+====
+include::macros:example$ui.adoc[tag=menu]
+====
+
+.Button macro
+[#ex-button]
+----
+include::macros:example$ui.adoc[tag=button]
+----
+
+.View result of <<ex-button>>
+[%collapsible.result]
+====
+include::macros:example$ui.adoc[tag=button]
+====
+
+== Attributes and substitutions
+
+.Attribute declaration and usage
+[#ex-attributes]
+----
+:url-home: https://asciidoctor.org
+:link-docs: https://asciidoctor.org/docs[documentation]
+:summary: Asciidoctor is a mature, plain-text document format for \
+       writing notes, articles, documentation, books, and more. \
+       It's also a text processor & toolchain for translating \
+       documents into various output formats (i.e., backends), \
+       including HTML, DocBook, PDF and ePub.
+:checkedbox: pass:normal[{startsb}&#10004;{endsb}]
+
+Check out {url-home}[Asciidoctor]!
+
+{summary}
+
+Be sure to read the {link-docs} too!
+
+{checkedbox} That's done!
+----
+
+.View result of <<ex-attributes>>
+[%collapsible.result]
+====
+// I have to use a nested doc hack here, otherwise the attributes won't resolve
+[.unstyled]
+|===
+a|
+:url-home: https://asciidoctor.org
+:link-docs: https://asciidoctor.org/docs[documentation]
+:summary: Asciidoctor is a mature, plain-text document format for \
+       writing notes, articles, documentation, books, and more. \
+       It's also a text processor & toolchain for translating \
+       documents into various output formats (i.e., backends), \
+       including HTML, DocBook, PDF and ePub.
+:checkedbox: pass:normal[{startsb}&#10004;{endsb}]
+
+Check out {url-home}[Asciidoctor]!
+
+{summary}
+
+Be sure to read the {link-docs} too!
+
+{checkedbox} That's done!
+|===
+====
+
+.Attribute assignment precedence (highest to lowest)
```

Comment:
I don't think the attribute precedence stuff belongs in the quick reference.

Reply:
I agree :+1:

Reply:
:+1: